inst/metadata/anc_run_options.json

```diff
@@ -3,19 +3,19 @@
   "description": "t_(OPTIONS_ANC_DESCRIPTION)",
   "controlGroups": [
     {
-      "label": "t_(OPTIONS_ANC_PREVALENCE_LABEL) - t_(OPTIONS_T1_LABEL)",
+      "label": "t_(OPTIONS_ANC_CLIENTS_LABEL)",
       "controls": [
         {
-          "name": "anc_prevalence_year1",
-          "label": "t_(OPTIONS_T1_LABEL)",
+          "name": "anc_clients_year2",
+          "label": "t_(OPTIONS_T2_LABEL)",
           "type": "select",
-          "helpText": "t_(OPTIONS_T1_HELP)",
+          "helpText": "t_(OPTIONS_T2_HELP)",
           "options":
-            <+anc_prevalence_year1_options+>,
+            <+anc_prevalence_year2_options+>,
           "required": false
```

Comment:
Also intended that this isn't required but the entry in the same row as this is required?

Reply:
I agree it looks weird. But the logic is that it's not required to specify a year for calibrating this. But if a year is specified, then the number of months is required to be specified.  This was the only way I could think to encode that?